pages/docs/community/glossary.md

```diff
@@ -0,0 +1,105 @@
+An AsyncPI document is a file that defines and annotates the different components of a specific Event-Driven API.
+
+
+## Bindings
+A "binding" (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST define protocol-specific information only.
+
+
+## Consumer
+In an Event Driven Architecture (EDA), a consumer is an application that listens for a particular event from a broker and reacts to it.
+
+
+## EDA
+EDA stands for Event-Driven Architecture, an architectural pattern that structures software systems around event production, detection, and consumption. In an event-driven architecture, components or services communicate by producing and consuming events representing significant changes or occurrences in the system.
+
+
+## Event
+An event is a message that provides details of something that has already occurred.
+
+An Event-Driven Architecture (EDA) uses events to trigger and communicate between services and is common in modern applications built with microservices.
+
+
+## Identifier
+This field represents a unique universal identifier of the application the AsyncAPI document defines. It must conform to the URI format, according to RFC3986.
+
+
+## Message-driven APIs
+Message-driven APIs, also known as Messaging APIs, facilitate communication between different components or services by exchanging messages. Instead of traditional request-response patterns in synchronous APIs, message-driven APIs rely on asynchronous messaging to enable decoupled and loosely coupled communication.
+
+
+## Microservices
+Microservices are a software architecture approach where applications are divided into small, independent services that communicate with each other through APIs. Each microservice focuses on a specific function, allowing scalability, agility, fault isolation, and technology flexibility.
+
+
+## Modelina
+Modelina is a library for generating data models based on inputs such as AsyncAPI, OpenAPI, or JSON Schema documents.
+
+
+## Producer
+A producer is an application that senses state changes (events) and publishes those events as messages. An event indicates a state change or update triggered by a user's/device's action.
+
+
+## Protocol
+A protocol is a set of rules that specifies how information is exchanged between applications and/or servers.
+
+
+## ProtocolVersion
+The version of the protocol used for the connection.
+
+
+## Studio
+Also known as the AsyncAPI studio is a tool that allows you to develop an AsyncAPI document, validate it, preview it, convert it to the latest version, and visualise event flows.
+
+
+
+
+<!---
+TODO: I need help deciding what these terms could be fit for and will appreciate any help or suggestions I can get.
+
+AsyncAPI specification
+AsyncAPI spec
+AsyncAPI file
+AsyncAPI specification file
+AsyncAPI instance
+AsyncAPI definition
+AsyncAPI contract
+
+Secondly, in writing AsyncAPI, are there any variants? like these:
+
+-  asyncAPI
+-  Async API
+
```

Comment:
Yes! Many people write it INCORRECTLY. 😂 

The only correct way to write our oss project name is `AsyncAPI`.

Reply:
Ok. Done this. 
We can highlight this information in an "important" callout.

<img width="658" alt="image" src="https://github.com/asyncapi/website/assets/33583060/70ab7f14-c027-4881-bd82-c0ef0073ad2f">

Let me know if this is something we want to do or if there are other ways to make this obvious; that'll be great too. Also, I'll like to know what flavour of Markdown we use for writing docs. It's my first time using the "Remember" tag. I kept spelling it with low caps "r" and wondered what I did wrong.

Reply:
Sure! A remember block could make great sense since this is a VERY common mistake :D